x-pack/plugins/security_solution/public/detections/pages/detection_engine/detection_engine.tsx

```diff
@@ -217,7 +217,7 @@ const DetectionEnginePageComponent: React.FC<DetectionEngineComponentProps> = ({
     [formatUrl, navigateToUrl]
   );
 
-  const alertsHistogramDefaultFilters = useMemo(
+  const alertsDefaultFilters = useMemo(
```

Comment:
Q: there is an already existing variable called `alertsTableDefaultFilters`, is there a difference between `alertsDefaultFilters` and that one?

Reply:
`alertsDefaultFilters` includes filters but`alertsTableDefaultFilters` does not
```
const alertsDefaultFilters = useMemo(
    () => [
      ...filters,
      ...buildShowBuildingBlockFilter(showBuildingBlockAlerts),
      ...buildThreatMatchFilter(showOnlyThreatIndicatorAlerts),
      ...(alertPageFilters ?? []),
    ],
    [filters, showBuildingBlockAlerts, showOnlyThreatIndicatorAlerts, alertPageFilters]
  );

  // AlertsTable manages global filters itself, so not including `filters`
  const alertsTableDefaultFilters = useMemo(
    () => [
      ...buildShowBuildingBlockFilter(showBuildingBlockAlerts),
      ...buildThreatMatchFilter(showOnlyThreatIndicatorAlerts),
      ...(alertPageFilters ?? []),
    ],
    [showBuildingBlockAlerts, showOnlyThreatIndicatorAlerts, alertPageFilters]
  );
```